### DISCUSSION TITLE
Дублирование таблиц в цикле

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#250] [Как это сделать?] [answered]
Здравствуйте, подскажите пожалуйста, как мне сделать, что бы нажав кнопку в основной таблице циклов, таблица в цикле продублировалась со всеми значениями?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Для пользователя нет дублирования таблиц — тк эта практика из Excel и здесь не применима.

Если вам нужно как Создателю сдублировать таблицу для ускорения разработки (что бы не набирать аналогичный набор полей) то это делается по правой кнопке на `name` таблицы в цикле (которую надо сдублировать) в таблице `Список таблиц`. Таблица дублируется без значений.

Если вам нужно продублировать цикл — то это доступно и для пользователей: https://ru.docs.totum.online/duplicate
